### PR TITLE
Pass service tracer to the yarpc dispatcher

### PR DESCRIPTION
**Pull request by madhuravi**
Fixes GFM-8

### CODE REVIEW

**Comment by madhuravi** [comment]
@uber-go/service-framework

**Comment by anuptalwalkar** [comment]
Can you update the description with just GFM-8 ? Getting paranoid about internal links:)

**Comment by madhuravi** [comment]
You're right Anup! I wasn't thinking.

File: modules/modules.go [hunk 1]
```diff
@@ -76,3 +79,8 @@ func (mb ModuleBase) Type() string {
 func (mb ModuleBase) Name() string {
 	return mb.name
 }
+
+// Tracer returns the module's service tracer
+func (mb ModuleBase) Tracer() opentracing.Tracer {
```
Comment (madhuravi) [inline modules/modules.go]: it's used in the same PR. Check yarpc.go

Comment (anuptalwalkar) [inline modules/modules.go]: oh cool. I missed.

**Pull request merged**